codeclimate-pylint

```diff
@@ -32,8 +32,14 @@ for path in include_paths:
         parameters.append(path)
 
 plugins_config = config.get('plugins', [])
+settings_module = config.get('settings_module')
```

Comment:
I think it's important to use `django_settings_module` so it doesn't get confused with possible other settings modules from other plugins

Reply:
I like this one 👍🏼